release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -2427,6 +2470,45 @@ Workaround: Manually or by using a daemon set, modify the file `/etc/cni/tuning/
 
+[id="ocp-4-13-ran-known-issues"]
+* If you specify an invalid subscription channel in the subscription policy that you use to perform a cluster upgrade, the {cgu-operator-first} indicates that the upgrade is successful immediately after {cgu-operator} enforces the policy because the `Subscription` resource remains in the `AtLatestKnown` state.
+(link:https://issues.redhat.com/browse/OCPBUGS-9239[*OCPBUGS-9239*])
+
+* `kdump` fails to generate the `vmcore` crash dump file on HPE Edgeline e920t and HPE ProLiant DL110 Gen10 servers with Intel E810 NIC and ice driver installed.
+(link:https://issues.redhat.com/browse/RHELPLAN-138236[*RHELPLAN-138236*])
+
+* In {ztp}, when you provision a managed cluster that contains more than a single node using a `SiteConfig` CR, disk partition fails when one or more nodes has a `diskPartition` resource configured in the `SiteConfig` CR.
+(link:https://issues.redhat.com/browse/OCPBUGS-9272[*OCPBUGS-9272*])
+
+* In clusters configured with PTP boundary clocks (T-BC) and deployed vDU applications, sometimes messages are not sent from the follower interface of the T-BC on the vDU for periods of up to 40 seconds. The rate or errors in the logs can vary. An example error log is below:
++
+.Example output
+[source,terminal]
+----
+2023-01-15T19:26:33.017221334+00:00 stdout F phc2sys[359186.957]: [ptp4l.0.config] nothing to synchronize
+----
+(link:https://issues.redhat.com/browse/RHELPLAN-145492[*RHELPLAN-145492*])
+
+* RFC2544 performance tests show that the `Max delay` value for a packet to traverse the network is over the minimum threshold. This regression is found in {product-title} 4.13 clusters running the Telco RAN DU profile.
+(link:https://issues.redhat.com/browse/OCPBUGS-13224[*OCPBUGS-13224*])
+
+* When you install a {sno} cluster using {ztp} and configure PTP and bare-metal events with HTTP transport, the `linuxptp-daemon` daemon pod intermittently fails to deploy.
+The required `PersistentVolumeClaim` (`PVC`) resource is created but is not mounted in the pod.
+The following volume mount error is reported:
++
+.Example output
+[source,terminal]
+----
+mount: /var/lib/kubelet/plugins/kubernetes.io/local-volume/mounts/local-pv-bc42d358: mount(2) system call failed: Structure needs cleaning.
+----
+To workaround the issue, delete the `cloud-event-proxy-store-storage-class-http-events` `PVC` CR and re-deploy the PTP Operator.
+(link:https://issues.redhat.com/browse/OCPBUGS-12358[*OCPBUGS-12358*])
+
+* `OSLAT` performance tests run on a {sno} cluster running {product-title} 4.13 show a max latency result greater than 20 microseconds.
+(link:https://issues.redhat.com/browse/RHELPLAN-155443[*RHELPLAN-155443*])
+
+* `cyclictest` performance tests run on a {sno} cluster running {product-title} 4.13 show a max cyclictest latency result greater than 20 microseconds.
```

Comment:
Per IBM Style:
> Do not start a sentence with a lowercase term; rewrite the sentence.

Further, the linked Jira issue points to `oslat` and not `cyclictest`. Based on [Measuring latency](https://docs.openshift.com/container-platform/4.12/scalability_and_performance/cnf-performing-platform-verification-latency-tests.html#cnf-measuring-latency_cnf-latency-tests), I understand that these are different tests. Moreover, the preceding point addresses the issue for `oslat`. Kindly verify.

Reply:
Updated